tests/utils/minio.go

```diff
@@ -35,6 +35,11 @@ import (
 	"github.com/cloudnative-pg/cloudnative-pg/pkg/postgres"
 )
 
+const (
+	minioImage       = "minio/minio:RELEASE.2020-04-23T00-58-49Z"
+	minioClientImage = "minio/mc:RELEASE.2022-06-11T21-10-36Z"
```

Comment:
Is there a reason why we are not also upgrading minioImage to a recent version?

Reply:
@mnencia during the testing, I only found that sometimes mc client leaves in hang status. let me upgrade both and run e2e again.

Reply:
upgraded and rerun a test case.